pkg/reconciler/ingress/ingress.go

```diff
@@ -362,39 +357,46 @@ func (c *Controller) dnsRecordFromIngress(ctx context.Context, ingress *networki
 			BlockOwnerDeletion: pointer.Bool(true),
 		},
 	})
-	return record, nil
-}
 
-func (c *Controller) endpointsFromIngress(ctx context.Context, ingress *networkingv1.Ingress) ([]*v1.Endpoint, error) {
-	var endpoints []*v1.Endpoint
+	return c.setEndpointsFromIngress(ctx, ingress, dnsRecord)
```

Comment:
re. https://github.com/Kuadrant/kcp-glbc/pull/52/files#r837228106
```suggestion
        //Set health check annotation
	return c.setEndpointsFromIngress(ctx, ingress, dnsRecord)	
	if healthEndpoint, ok := ingress.Annotations[dns.HealthEndpointAnnnotation]; ok {
		dnsRecord.Annotations[dns.HealthEndpointAnnnotation] = healthEndpoint
	}

	return c.setEndpointsFromIngress(ctx, ingress, dnsRecord)
```